### PR TITLE
CONSOLE-3972: add Last Status to Pod details Container list and Conta…

### PR DESCRIPTION
**Pull request by rhamilto**
…iner details

Pod details page Containers list at various breakpoints

https://github.com/openshift/console/assets/895728/3fee3ae5-1faa-44ca-aafb-22fef57bac67

Container details page
localhost_9000_k8s_ns_knative-serving-ingress_pods_3scale-kourier-gateway-79bc5c58d9-c2lw7_containers_kourier-gateway

Empty state
localhost_9000_k8s_ns_knative-serving-ingress_pods_3scale-kourier-gateway-79bc5c58d9-c2lw7_containers_kourier-gateway (1)
localhost_9000_k8s_ns_knative-serving-ingress_pods_3scale-kourier-gateway-79bc5c58d9-c2lw7_containers_kourier-gateway (2)

### CODE REVIEW

**Comment by yanpzhan** [comment]
@rhamilto I tested the pr and added a case for it, could you help to review the case? Thanks! case link: https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-72714

**Comment by yanpzhan** [comment]
@rhamilto The pr still has work-in-progress label, will there be other update in code?

**Comment by rhamilto** [comment]
> @rhamilto The pr still has work-in-progress label, will there be other update in code?

@yanpzhan, I don't expect there will be changes, but this PR is waiting on code approval from @jhadvig who has been out on PTO.

File: frontend/public/components/pod.tsx
Comment (jhadvig) [inline frontend/public/components/pod.tsx:533]: I would rather have the render logic be part of a single component rather then three separate ones, since they are not used anywhere.

**Review by jhadvig — COMMENTED** [review]
The change looks right to me. Adding one small comment.

File: frontend/public/components/pod.tsx
Comment (rhamilto) [reply frontend/public/components/pod.tsx:533]: The separate components are necessary since the data is optional

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**
